### ISSUE TITLE
Node delivers only 0 values

### ISSUE THREAD

**Issue by Joehne** [dehsgr/node-red-contrib-alphaess#4]
Hello,
due to problems with the login at alphaess I have now installed version 1.6.0. I use the backup data because I always received 0 values for real-time data. In the new version 1.6.0 the backup data are now also all 0.
Thank you very much,
Greetings Jürgen

**Comment by dehsgr** [comment]
@Joehne that sounds very strange. I cannot see anything not working here. Can you enable verbose logging in node-red and post a log here? Maybe we find out what's going on...

**Comment by Joehne** [comment]
I have set the level to"debug" in the settings file.
The Output istonly:
9 Oct 18:04:51 - [debug] [alphaess-monitoring:Alpha S5] Fetching realtime data (backup path)...
9 Oct 18:04:52 - [debug] [alphaess-monitoring:Alpha S5] Processing data...
9 Oct 18:05:03 - [debug] [alphaess-monitoring:Alpha S5] Fetching realtime data (backup path)...
9 Oct 18:05:04 - [debug] [alphaess-monitoring:Alpha S5] Processing data...

**Comment by dehsgr** [comment]
Strange. Can you untick backup path checkbox and publish flow? Afterwards re-tick it and publish flow again? Does this change sth.?

**Comment by Joehne** [comment]
here the log without backup

9 Oct 18:34:29 - [debug] [alphaess-monitoring:Alpha S5] Fetching realtime data...
9 Oct 18:34:29 - [warn] [alphaess-monitoring:Alpha S5] There was an error fetching realtime data for AK1510120700115: Malformed or empty response!
9 Oct 18:34:29 - [debug] [alphaess-monitoring:Alpha S5] Processing data...
9 Oct 18:34:29 - [error] [function:prepare] TypeError: Cannot read property 'toFixed' of undefined
9 Oct 18:34:41 - [debug] [alphaess-monitoring:Alpha S5] Fetching realtime data...
9 Oct 18:34:41 - [warn] [alphaess-monitoring:Alpha S5] There was an error fetching realtime data for AK1510120700115: Malformed or empty response!
9 Oct 18:34:41 - [debug] [alphaess-monitoring:Alpha S5] Processing data...
9 Oct 18:34:41 - [error] [function:prepare] TypeError: Cannot read property 'toFixed' of undefined

by the way: See a PM at Storion4you

**Comment by Joehne** [comment]
now I just see that I no longer see any current data on the Alpha Portal either. In this respect, I assume for the time being that the node is working properly.
Alpha-Ess is the problem :(

**Comment by Joehne** [comment]
However, the battery state of charge is still displayed. The value would be enough for me.

**Comment by Joehne** [comment]
Strange.
Now the display in the portal is ok again, but the node still delivers 0 values.

**Comment by dehsgr** [comment]
I would like you to send me some test credentials to your system to dehsgr at-sign outlook dot com. So I could check this from my site. Alpha ESS seems to update/change sth. at their servers.

**Comment by Joehne** [comment]
have sent the link for guest access.

**Comment by dehsgr** [comment]
Thank you very much. The web portal function (this is the one I use to query the data) gives me only zero values too. There must be sth. wrong on Alpha ESS system site. I'm not able to do sth. from my site here. :-(

**Comment by neiloakley** [comment]
Unsure if it's the same, but in the last 24-48 hours i'm also facing issues. Log from node shows this:

There was an error during login operation into Alpha ESS monitoring portal: SyntaxError: Unexpected token  in JSON at position 0

We got the following unprocessable body:
�ER�r�0|��8h��6��<ƀ@7"��q���;�����������o��v5�&$mJ����P4Y+,��%Ih�	7B�ca�&�f�km��VN̤��G(:�`�*����`ci���-5�D�`TuDڞ�(�D����.Bb˶ up$���R_Q�+H�`8H�2��)�nO�#<��iS;a�<�p���GGj{#X����V;��#ou� E�K��ť/����)M��
��Wl��׊b�}��d�����.�W�g;F΅Ls��}#_�e�أ�q?ڟ::���Z���3>p�������Τ��r��ݕ��4\�,���c�@��z@�9������=�91���͞�[��
�H�D�7ct��;��Ij����)#�{w�CEWŮ4$��ǝ�s|��?�Y����Կ�������Z��u�N�=2؀[���Ʃ���	����6벅/yՍ_+�ջ�Bc��g���Dy����ny+��mN4>S��yW	Hҝ�/������R���.��z>�<сsAB�{%����>h�y���R�)��	����7�

**Comment by dehsgr** [comment]
@Joehne I recently published another fix reflecting backup path. You might get values again now. Would be happy for your feedback.

**Comment by Joehne** [comment]
I opened a ticket with Alpha on 9 October. After some discussions, Alpha actually managed to display the data in the portal again. I can now access the data directly and also via the backup path. This happened two days ago and I only noticed the correction yesterday. I am still using version 1.7.0.  

Upgrade to 1.7.1:
Again, the original and backup path works. 

Thank you for the info and for your work!
